src/pytopmod/core/dlfl/primitives.py

```diff
@@ -0,0 +1,37 @@
+"""Convenience functions to create primitive DLFL Meshes."""
+from pytopmod.core.dlfl import operators
+from pytopmod.core.dlfl.mesh import DLFLMesh
+
+
+def triangle() -> DLFLMesh:
+    """Creates and returns a triangle with two faces."""
+    mesh = DLFLMesh()
+
+    operators.create_point_sphere(mesh, (1.0, 1.0, 1.0))
```

Comment:
nit: I am not a big fan of shortcutting the indices.

```python
  v1, f1 = operators.create_point_sphere(mesh, (1.0, 1.0, 1.0))
  v2, f2 = ....
  ...
   f4, f5 = operators.insert_edge(mesh, v1, f1, v2, f2)
  ...
```

Reply:
Agreed. Applied suggestion (used variable names with _ to make the linter happy).